src/definitions_cxx.hpp

```diff
@@ -990,6 +993,14 @@ enum GridDefaultActiveMode : uint8_t {
 	GridDefaultActiveModeMaxElement // Keep as boundary
 };
 
+enum SliceModeDefault  {
+	SliceModedefaultCut,
```

Comment:
Nit 1: don't need to write SliceModedefault as a prefix to Cut, Once, Stretch, Loop, MaxElement because when you refer to this enum elsewhere it will be referred to as SliceModeDefault::Cut

Nit 2: make Cut, Once, Stretch, Loop capitals (CUT, ONCE, STRETCH, LOOP)

Nit 3: don't need max element here. (There's no max element in the SampleRepeatMode enum above)